ffi/Makefile

```diff
@@ -14,22 +14,31 @@ ifeq ($(OS),Windows_NT)
     endif
 	OS_EXT=dll
 else
-	UNAME_OS := $(shell uname -s)
-	ifeq ($(UNAME_OS),Darwin)
-		OS_FLAG=darwin
-	endif
-	ifeq ($(UNAME_OS),Linux)
-		OS_FLAG=linux
+	ifeq ($(strip $(OS_FLAG)),)
```

Comment:
OS_FLAG here vs OS further above?

Reply:
again, copied in from the csharp branch (which includes readme updates to show OS_FLAG override from cli).
Not relevant to this PR